SYCL/ESIMD/api/functional/operators/operator_decrement_and_increment_accuracy_core.cpp

```diff
@@ -0,0 +1,45 @@
+//==------- operator_logical_not.cpp  - DPC++ ESIMD on-device test ---------==//
```

Comment:
nit: file name is not correct.

Reply:
this comment has been applied in this commit: 75b472db1fe0777cd84eee7aca25852f5a44d436